core/views.py

```diff
@@ -1 +1,18 @@
-# Create your views here.
+#from django.shortcuts import render
+#
+# # Create your views here.
+
+from django.shortcuts import get_object_or_404, render
+from django.http import HttpResponseRedirect, HttpResponse
+from django.core.urlresolvers import reverse
+from django.views import generic
+from django.utils import timezone
+
+from .models import Voter
+
+class IndexView(generic.DetailView):
```

Comment:
So the IndexView won't be a DetailView (because it doesn't show the Details for anything!) We can use a TemplateView here for now and then adjust that later to meet future needs.